### DISCUSSION TITLE
Ability to edit a goal and retain its historical identity

### DISCUSSION THREAD

**Discussion by carlgunderson** [plausible/analytics#2718] [Anything Else]
Hello,

It would be nice if goals were tied to some hidden ID rather than the name we give it, so that we could edit goal names.

**Comment by Hricha-Shandily** [comment]
You can now edit [Pageview](https://plausible.io/docs/pageview-goals#edit-a-pageview-goal) and [Custom Event](https://plausible.io/docs/custom-event-goals#edit-a-custom-event-goal) goals, including changing their settings and display names.
